src/main/java/uk/gov/companieshouse/mapper/email/DissolutionEmailMapper.java

```diff
@@ -81,4 +83,23 @@ public SignatoryToSignEmailData mapToSignatoryToSignEmailData(Dissolution dissol
 
         return signatoryToSignEmailData;
     }
+
+    public PendingPaymentEmailData mapToPendingPaymentEmailData(Dissolution dissolution) {
+        final String applicantEmailAddress = dissolution.getCreatedBy().getEmail();
+        final String dissolutionReferenceNumber = dissolution.getData().getApplication().getReference();
+        final String companyNumber = dissolution.getCompany().getNumber();
+        final String companyName = dissolution.getCompany().getName();
```

Comment:
I don't think there is any need to make these separate vaiables. Could save around 5 lines of code here